content/fr/about/_index.md

```diff
@@ -58,7 +58,7 @@ title: "À propos d'InnerSource Commons"
         <p>La communauté InnerSource Commons est ouverte à tous. Certains participants à la communauté vont plus loin et peuvent devenir membres de l'ISC. ISC est une organisation avec des membres adhérents&nbsp;: les membres de l'ISC jouent donc un rôle similaire à celui des actionnaires des sociétés cotées en bourse. </p>
         <p>Les membres ont des droits particuliers au sein de l'ISC&nbsp;; par exemple, ils peuvent voter aux élections du conseil d'administration et de gouvernance. Pour être éligible en tant que membre, une personne doit être nommée par un membre actuel. L'adhésion est purement fondée sur le mérite, gratuite et réservée aux personnes physiques.
         </p>
-        <a href="/about/members" class="btn-link">Liste des Membres d'InnerSource Commons<i class="ti-arrow-right"></i></a>
+        <a href="https://github.com/InnerSourceCommons/innersourcecommons.org/pull/481members" class="btn-link">Liste des Membres d'InnerSource Commons<i class="ti-arrow-right"></i></a>
```

Comment:
```suggestion
        <a href="/fr/about/members" class="btn-link">Liste des Membres d'InnerSource Commons<i class="ti-arrow-right"></i></a>
```
This might be right?

Reply:
fixed in last commit